README.md

```diff
@@ -12,22 +12,34 @@ Highly Available Tomcat 9 Image
        -> Update your application build.yaml to use this image for hosting your java application 
        -> Update your application deployment.yaml to add the rbac authorizations and roles required to run the tomcat in a cluster and don't forget to add the service account required to run your container under the spec.
 
-#### Note: To run with existing docker configuration, you will not be needed to make any changes to the docker folder
+#### Note: To run with existing docker configuration, you will not be need to make any changes to the docker folder
 
-# How to build image
-## In Openshift
+# Building & Deploying the Image
+## In Openshift (via developer CLI)
 ```
+We tag images using the name of the Jira Issue that caused the image to be built (the SUFFIX parameter).
+Per openshift requirements, we do this in lower-case.
+If you are just running an experiment, you can use your name, your idir, or 'build'.
+
 # NOTE: Switch to target namespace, or set `-n <namespace>` on all `oc` commands
-# oc project ywinub-tools
+# oc project perrsi-tools
 
 # Process template, and create OpenShift resources
-oc -n ywinub-tools process -f openshift/build.yaml SUFFIX=-cvarjao IMAGE_TAG_SUFFIX=-cvarjao | oc -n ywinub-tools create -f -
+oc -n perrsi-tools process -f openshift/build.yaml SUFFIX=oneteam-123 | oc -n perrsi-tools create -f -
+
+# After the first run, use oc update instad of oc create to update the existing image.
 
 # Start a new build using the current repository in the working directory as source
-oc -n ywinub-tools start-build bc/ha-tomcat9-cvarjao --from-repo=.
+oc -n perrsi-tools start-build bc/ha-tomcat9-oneteam-123 --from-repo=.
+
+# NOTE: This includes all commits done locally, but not yet pushed.
+#       If all commits are already pushed, there is no need for `--from-repo=.`
+
+# When making the image available in perrsi-prod, we update the -latest tag to this image, and update -stable if appropriate.
+# As this tag-over procedure is manual, we will apply a numerically-incrementing build number, as exposing Jira issues downstream is low-value.
 
-# NOTE: it includes all commits done locally, but not yet pushed
-#       if all commits are already pushed, there is no need for `--from-repo=.`
+oc tag perrsi-tools/ha-tomcat9:9.0.27-oneteam-123 perrsi-prod/ha-tomcat9:9.0.27-7 --reference-policy=local
+oc tag perrsi-tools/ha-tomcat9:9.0.27-oneteam-123 perrsi-prod/ha-tomcat9:9.0.27-latest --reference-policy=source
```

Comment:
I had had the -latest as source so that the next build of this image would update to -latest, but I'm happy to suggest manually tagging it over instead.